packages/ckeditor5-table/src/utils/ui/widget.js

```diff
@@ -49,11 +49,11 @@ function isTableWidget( viewElement ) {
 	return !!viewElement.getCustomProperty( 'table' ) && isWidget( viewElement );
 }
 
-function findAncestor( parentName, positionOrElement ) {
+function findAncestor( positionOrElement ) {
 	let parent = positionOrElement.parent;
 
 	while ( parent ) {
-		if ( parent.name === parentName ) {
+		if ( parent.is( 'element' ) && isTableWidget( parent ) ) {
```

Comment:
This change makes this helper no longer matching it's name (this should not be `findAncestor` or `isTableWidget` should not be called here).